SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/Build/Build.cs

```diff
@@ -232,15 +233,15 @@ public static void Main(string[] args)
                     Console.WriteLine("worker coordinator path did not exist");
                 }
 
-                var archiveFileName = "UnrealSimulatedPlayer@Linux.zip";
+                var archiveFileName = gameName + "SimulatedPlayer@Linux.zip";
```

Comment:
This will have a different archive file name than before the PR I believe, which means the warning above may be invalid, among other external assumptions (whatever may be using sim players)

Reply:
Good point will check in with @Zomtorg and @WVerlaek how we should approach this. we may actually just want to keep the old line thinking about it.

Reply:
Yeah, this change will break simulated player deployments indeed. The worker config that references this SimulatedPlayer archive is generated by the GDK and doesn't know about the name of the project/game it is in. With our current implementation it won't be trivial to change the generation of the worker config to include the game name, it would be preferable if the simplayer archive's name remains the same for each game/project